tests/test_workflow.py

```diff
@@ -1,13 +1,52 @@
 """Tests for entire workflows."""
 from pathlib import Path
 
+import matplotlib.colors
 import matplotlib.testing.compare
 import numpy
 import xarray as xr
 from click.testing import CliRunner
+from matplotlib import pyplot as plt
 
 from wam2layers.cli import cli
 
+try:  # Use cartopy if it's available
+    import cartopy.feature as cfeature
+    from cartopy import crs
+except ImportError:
+    crs = None
+    cfeature = None
+
+
+def plot_difference(
+    expected_output: xr.DataArray, output: xr.DataArray, fpath: Path
+) -> None:
+    """Plot the difference between the expected output and actual output."""
+    if not fpath.parent.exists():
+        fpath.parent.mkdir()  # Make figures directory if it does not exist yet
+
+    if crs is not None:
+        subplot_kw = dict(projection=crs.PlateCarree())
+    else:
+        subplot_kw = None
+
+    fig, ax = plt.subplots(
+        figsize=(16, 10),
+        subplot_kw=subplot_kw,
+    )
+    (expected_output - output).plot.pcolormesh(
```

Comment:
This also triggers pylance for me, though I'm not really sure why: `Argument missing for parameter "self"`

Reply:
No idea what is going wrong here. That again seems like a bugged type in pylance. Mypy does not detect this.